nestkernel/connection_manager.h

```diff
@@ -158,6 +158,8 @@ class ConnectionManager : public ManagerInterface
     size_t n,
     std::string syn_model );
 
+  void connect_sonata( const DictionaryDatum& graph_specs, const long chunk_size );
```

Comment:
Please add a doxygen comment explaining what dictionary is expected and what chunk_size is.